priv/device_mock/device.js

```diff
@@ -25,14 +25,24 @@ function loop() {
  */
 function readTelemetry() {
   return {
-    tmp: readTemperature()
+    tmp: readTemperature(),
+    hum: readHumidity(),
+    moist: readSoilMoisture()
   }
 }
 
 function readTemperature() {
   return randomNumber(10, 20)
 }
 
+function readHumidity() {
+  return randomNumber(30, 60)
+}
+
+function readSoilMoisture() {
+  return randomNumber(40, 80)
+}
+
 function randomNumber(min, max) {
   return Math.floor(Math.random() * (max - min) + min);
 }
```

Comment:
none of these functions use this for anything, why don't we use arrow functions for them?